src/Runner.Worker/action_yaml.json

```diff
@@ -88,12 +88,8 @@
         "input-default-context": {
             "context": [
                 "github",
-                "strategy",
```

Comment:
did you mean to do this?

Reply:
strategy and matrix are the ones i'm mainly concerned about removing

Reply:
I might keep `strategy` since it contains well-known keys fail-fast, job-index, job-total, and max-parallel.